src/rpc/server/scala/serv/src/runserver.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2021 Concurrent Technologies Corporation.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const os = require('os')
+const child_process = require('child_process')
+
+function runServer(scriptDir, args) {
+  let scriptName =
+    os.platform() === 'win32'
+      ? 'omega-edit-grpc-server.bat'
+      : 'omega-edit-grpc-server'
+
+  if (os.platform() !== 'win32') {
+    child_process.execSync(`chmod +x ${scriptDir}/${scriptName}`)
+  }
+
+  return child_process.spawn(scriptPath, args, {
+    cwd: scriptDir,
+    stdio: 'ignore',
+    detached: true,
+  })
+}
+
+module.exports = { runServer }
```

Comment:
Seems we can add `#!/usr/bin/env node` and `chmod +x` for non-Windows, but for Windows we need to call `node script.js`, making it the client's problem again.

Reply:
basically. We could `node script.js` on all oses and have it be fine but then the client would need to handle that

Reply:
Okay, I'm good with that!  The `server.ts` executes the entry point script.  It should do that since the client can manage the server process lifecycle (PID files, starting, stopping gracefully, killing, monitoring, etc).